documentation/AWS.md

```diff
@@ -0,0 +1,76 @@
+AWS IAM role and policy setup
+==========================
+
+IAM Policy Setup
+-----
+
+* Log onto AWS and navigate to the IAM policy panel
+![](screenshots/AWS/IAMPolicy1.png?raw=true)
+
+* Create a IAM Policy
+![](screenshots/AWS/IAMPolicy2.png?raw=true)
+
+* Write a name and description in for the policy, paste in the below policy document, then create the policy by clicking the button in the lower right corner.
```

Comment:
Write a name and description in for the policy -> Write a name and description for the policy 

(the 'in' can be removed)